Misc/NEWS.d/next/Library/2018-07-29-13-50-32.bpo-32321.hDoNKC.rst

```diff
@@ -0,0 +1 @@
+Add pure Python fallback for functools.reduce
```

Comment:
Add a sentence ending period and "Path by *yourname*."

Reply:
Sorry, what do you mean?

Reply:
Thanks, the typo confused me.

Reply:
Oh, sorry. I just noticed it!